tests/chainer_tests/links_tests/loss_tests/test_negative_sampling.py

```diff
@@ -4,27 +4,28 @@
 
 import chainer
 from chainer import backend
-from chainer.backends import cuda
+from chainer.backends import _cpu
```

Comment:
How about using `chainer.backend.CpuDevice.send` for cpu transfer preferring public APIs?